sdks/techbooks-ts/src/lib/security.ts

```diff
@@ -6,7 +6,7 @@ import * as components from "../models/components";
 
 export enum SecurityErrorCode {
     Incomplete = "incomplete",
-    UnrecognisedSecurityType = "unrecognized_security_type",
+    UnrecognizedSecurityType = "unrecognized_security_type",
```

Comment:
This is probably also an api break. Again, I don't expect it to be accepted, but...